pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -1268,6 +1271,74 @@ var _ = Describe("Manager", func() {
 				DedicatedCPUPlacement: true}, "false", "off"),
 			Entry("disabled if vmi has the disable free page reporting annotation", nil, false, nil, "true", "off"),
 		)
+
+		It("should return SEV platform info", func() {
+			sevNodeParameters := &api.SEVNodeParameters{
+				PDH:       "AAABBBCCC",
+				CertChain: "DDDEEEFFF",
+			}
+
+			mockConn.EXPECT().GetSEVInfo().Return(sevNodeParameters, nil)
+
+			manager, _ := NewLibvirtDomainManager(mockConn, testVirtShareDir, testEphemeralDiskDir, nil, "/usr/share/OVMF", ephemeralDiskCreatorMock, metadataCache)
+			sevPlatfomrInfo, err := manager.GetSEVInfo()
+			Expect(err).ToNot(HaveOccurred())
+			Expect(sevPlatfomrInfo.PDH).To(Equal(sevNodeParameters.PDH))
+			Expect(sevPlatfomrInfo.CertChain).To(Equal(sevNodeParameters.CertChain))
+		})
+
+		It("should return a VirtualMachineInstance launch measurement", func() {
+			domainLaunchSecurityParameters := &libvirt.DomainLaunchSecurityParameters{
+				SEVMeasurementSet: true,
+				SEVMeasurement:    "AAABBBCCC",
+			}
+			loaderBytes := []byte("OVMF binary with SEV support")
+			vmi := newVMI(testNamespace, testVmName)
+
+			mockConn.EXPECT().LookupDomainByName(testDomainName).Return(mockDomain, nil)
+			// Make sure that we always free the domain after use
+			mockDomain.EXPECT().Free()
+			mockDomain.EXPECT().GetLaunchSecurityInfo(uint32(0)).Return(domainLaunchSecurityParameters, nil)
+
+			ovmfDir, err := os.MkdirTemp("", "ovmfdir")
+			Expect(err).ToNot(HaveOccurred())
+			defer os.RemoveAll(ovmfDir)
+			err = ioutil.WriteFile(filepath.Join(ovmfDir, efi.EFICodeSEV), loaderBytes, 0644)
+			Expect(err).ToNot(HaveOccurred())
+
+			manager, _ := NewLibvirtDomainManager(mockConn, testVirtShareDir, testEphemeralDiskDir, nil, ovmfDir, ephemeralDiskCreatorMock, metadataCache)
+			sevMeasurementInfo, err := manager.GetLaunchMeasurement(vmi)
+			if runtime.GOARCH == "amd64" {
```

Comment:
Shouldn't we skip all the suite if this isn't amd64?

Reply:
I am not sure about skipping, since it may hide potential issues. If the code is expected to error-out on non-amd64, IMHO, better to have a test in place to ensure it. In this particular case, `manager.GetLaunchMeasurement` fails due to missing ovmf code binary with sev support.

Reply:
Ok, fine to me!